code/modules/mining/kinetic_crusher.dm

```diff
@@ -1,52 +1,89 @@
 /*********************Mining Hammer****************/
 /obj/item/kinetic_crusher
+	name = "proto-kinetic crusher"
+	desc = "An early design of the proto-kinetic accelerator, it is little more than an combination of various mining tools cobbled together, forming a high-tech club. \
+	While it is an effective mining tool, it did little to aid any but the most skilled and/or suicidal miners against local fauna."
 	icon = 'icons/obj/mining.dmi'
 	icon_state = "crusher"
 	item_state = "crusher0"
 	item_icons = list(
 		slot_l_hand_str = 'icons/mob/inhands/weapons/hammers_lefthand.dmi',
 		slot_r_hand_str = 'icons/mob/inhands/weapons/hammers_righthand.dmi'
 		)
-	name = "proto-kinetic crusher"
-	desc = "An early design of the proto-kinetic accelerator, it is little more than an combination of various mining tools cobbled together, forming a high-tech club. \
-	While it is an effective mining tool, it did little to aid any but the most skilled and/or suicidal miners against local fauna."
-	force = 0 //You can't hit stuff unless wielded
-	w_class = WEIGHT_CLASS_BULKY
+	w_class = ITEMSIZE_HUGE
 	slot_flags = SLOT_BACK
 	throwforce = 5
 	throw_speed = 4
-/*
-	armour_penetration = 10
-	custom_materials = list(/datum/material/iron=1150, /datum/material/glass=2075)
-*/
 	hitsound = 'sound/weapons/bladeslice.ogg'
+	drop_sound = 'sound/items/drop/sword.ogg'
+	pickup_sound = 'sound/items/pickup/sword.ogg'
 	attack_verb = list("smashed", "crushed", "cleaved", "chopped", "pulped")
 	sharp = TRUE
 	edge = TRUE
-	// sharpness = SHARP_EDGED
 	action_button_name = "Toggle Light"
-	// actions_types = list(/datum/action/item_action/toggle_light)
-	// var/list/trophies = list()
+	var/requires_wield = TRUE
```

Comment:
why are you moving the commented code up and removing the comments?

Reply:
This was probably accidental, from when I was trying to convert these to work as proper two-handed weapons. I'll restore these lines.